ietf/ietfauth/views.py

```diff
@@ -491,8 +491,11 @@ def password_reset(request):
             if not user:
                 # try to find user ID from the email address
                 email = Email.objects.filter(address=submitted_username).first()
-                if email and email.person and email.person.user:
-                    user = email.person.user
+                if email and email.person:
+                    if email.person.user:
+                        user = email.person.user
+                    else: 
+                        send_account_creation_email(request, email.email_address())
```

Comment:
I've manually asserted that if I remove these lines, then the test case fails too.